unlock-app/src/__tests__/components/creator/CreatorLock.test.js

```diff
@@ -10,6 +10,8 @@ jest.mock('next/link', () => {
   return ({ children }) => children
 })
 
+const toggleEditing = jest.fn()
```

Comment:
```suggestion
```

let's not do this. Since you're not testing the toggle result in this test, we don't need this